main.go

```diff
@@ -7,19 +7,23 @@ import (
 	"strings"
 
 	"github.com/drone-plugins/drone-git-push/repo"
-	"github.com/drone/drone-go/drone"
-	"github.com/drone/drone-go/plugin"
+	"github.com/drone/drone-plugin-go/plugin"
 )
 
 var (
 	buildCommit string
 )
 
+const (
+	DEFAULT_REMOTE_NAME = "deploy"
```

Comment:
I've been looking at protobuf constant adaptations too long - fixed in the new commit.